dist_tables/ddl.rst

```diff
@@ -129,6 +129,19 @@ To co-locate a number of tables, distribute one and then put the others into its
 
 Information about co-location groups is stored in the :ref:`pg_dist_colocation <colocation_group_table>` table, while :ref:`pg_dist_partition <partition_table>` reveals which tables are assigned to which groups.
 
+Using the functions above will distribute tables with the desired co-location. However tables which were distributed using Citus version 5 lack the appropriate entries in :code:`pg_dist_partition`. Citus will refuse to run features like :ref:`dist_agg` on tables not explicitly marked as co-located, even if their shards happen to be placed correctly.
```

Comment:
A few notes:

- Should we make this a separate sub-section? (Something like "Upgrading from Citus 5.x" ?)

- We can give a bit more context: i.e., "Start with Citus 6.0, we made co-location a first-class concept, and started tracking this info in pg_dist_colocation. Since Citus 5.x didn't have this concept, tables created with Citus 5 won't be explicitly marked in metadata, despite being physically co-located. 
Since Citus uses this co-location metadata information for query optimization and pushdown, it becomes critical to inform Citus of this co-location for previously created tables. To do this..."